client/src/components/infoDrawer/infoDrawer.js

```diff
@@ -49,14 +80,46 @@ class InfoDrawer extends PureComponent {
         onClose={this.handleClose}
         {...{ isOpen, position }}
       >
-        <InfoFormat
-          {...{
-            datasetTitle,
-            aboutURL,
-            singleValueCategories,
-            dataPortalProps,
-          }}
-        />
+        <Async
+          watchFn={InfoDrawer.watchAsync}
+          promiseFn={this.fetchAsyncProps}
+          watchProps={{ schema }}
```

Comment:
Will the schema ever change?  There's a minor edge case where a user clips to a single continuous value, how will the drawer react?